manifests/z/Ziad/TCG/6.9-stable/Ziad.TCG.locale.en-US.yaml

```diff
@@ -0,0 +1,48 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Ziad.TCG
+PackageVersion: 6.9-stable
+PackageLocale: en-US
+Publisher: Ziad (Mr.X) Software
+PublisherUrl: https://insertx2k.github.io/mrx
+PublisherSupportUrl: https://insertx2k.github.io/mrx
+PrivacyUrl: https://insertx2k.github.io/temp_cleaner_gui/privacy.html
+Author: Ziad (Mr.X) Software
+PackageName: Temp_Cleaner GUI
+PackageUrl: https://insertx2k.github.io/temp_cleaner_gui
+License: GPL-2.0-or-later
+LicenseUrl: https://raw.githubusercontent.com/InsertX2k/temp_cleaner_gui/main/LICENSE
+Copyright: Copyright (c) Ziad Ahmed (Mr.X)
+# CopyrightUrl:
+ShortDescription: A Free and Open-Source program to free up disk space on your PC.
+Description: Temp_Cleaner GUI is yet another free and open-source piece of software made with your comfortability in mind, What makes Temp_Cleaner GUI different is that it respects your privacy and doesn't ask for a subscription unlike others, our program doesn't collect any sort of personal identification data and it has a lot of cleaning options that aren't available in many popular Free alternatives to this program.
+Moniker: tcg
+Tags:
+- ccleaner
+- cleaner
+- gui
+- optimize
+- pc
+- pc-cleaner
+- speed
+- temp
+- temp_cleaner
+ReleaseNotes: |-
+  What is new in version 6.9-stable?
+	1 Fixed the issue with the Updater module that causes it to prevent Temp_Cleaner GUI from starting until it finishes checking for updates.
+	2 Improved the overall behavior of the Updater window & module.
+	3 Added a 25 seconds timeout for the check for updates function (to prevent the issue of the program staying in the background for too long)
+	4 Improved overall program exception handling
+	5 Improved the way Temp_Cleaner GUI displays and handles errors.
+	6 Major bug fixes and improvements to the behavior of some cleaners.
+	7 Minor improvements in overall code readability.
+	8 Now the Legacy Microsoft Edge Windows Store Version cleaner is automated! (I know this should have been automated since the first release of Temp_Cleaner GUI, I am sorry I forgot about it ;-;)
+	Please note that not all new features and bug fixes are listed here, and you may see other improvements/or new features that aren't listed here.
+
```

Comment:
```suggestion
  What is new in version 6.9-stable?
  1 Fixed the issue with the Updater module that causes it to prevent Temp_Cleaner GUI from starting until it finishes checking for updates.
  2 Improved the overall behavior of the Updater window & module.
  3 Added a 25 seconds timeout for the check for updates function (to prevent the issue of the program staying in the background for too long)
  4 Improved overall program exception handling
  5 Improved the way Temp_Cleaner GUI displays and handles errors.
  6 Major bug fixes and improvements to the behavior of some cleaners.
  7 Minor improvements in overall code readability.
  8 Now the Legacy Microsoft Edge Windows Store Version cleaner is automated! (I know this should have been automated since the first release of Temp_Cleaner GUI, I am sorry I forgot about it ;-;)
  Please note that not all new features and bug fixes are listed here, and you may see other improvements/or new features that aren't listed here.
```

Reply:
YAML doesn't like tabs, and I believe requires indentation to be represented by 2 spaces.

Reply:
Thanks for the explaination.